github/Organization.py

```diff
@@ -596,6 +597,31 @@ def has_in_public_members(self, public_member):
         )
         return status == 204
 
+    def invite_to_organization(self, user, admin=None):
+        """
+        :calls: `PUT /orgs/:org/memberships/:username`_
```

Comment:
Would you mind adding the GitHub link to this endpoint?

Reply:
Not a problem. Sorry I missed it.

Reply:
Let's get this merged soon 😄